fifo.js

```diff
@@ -33,7 +36,9 @@ class FIFO {
         this._throwIfReader()
 
         let cmd = this._generateReadCommand()
-        return cp.execSync(cmd).toString()
```

Comment:
Note that the toString() here was only necessary because *for some reason* the default value of encoding is 'utf8' for exec, but 'buffer' for execSync. Now that we are explicitly providing the encoding it's no longer necessary.